guides/release/tutorial/deploying.md

```diff
@@ -68,6 +72,34 @@ surge dist funny-name.surge.sh
 Note we are building with the Google maps API key as shown above for UNIX platforms.
 For windows you will need to set the variable according the example in the previous section.
 
+### ZEIT Now
+
+[ZEIT Now](https://zeit.co) is a cloud platform for websites and serverless APIs, that you can use to deploy your Ember projects to your personal domain (or a free `.now.sh` suffixed URL).
+
+This guide will show you how to get started in a few quick steps:
+
+#### Step 1: Installing Now CLI
+
+To install their command-line interface with [npm](https://www.npmjs.com/package/now), run the following command:
+
+```bash
+npm install -g now
+```
+
+#### Step 2: Deploying
+
+You can deploy your application by running the following command in the root of the project directory:
+
+```bash
+now
+```
+
+**Alternatively**, you can also use their integration for [GitHub](https://zeit.co/github) or [GitLab](https://zeit.co/gitlab).
+
+That’s all!
+
+Your site will deploy, and you will receive a link similar to the following: https://ember.now-examples.now.sh
```

Comment:
```suggestion
Your site will deploy, and you will receive a link similar to the following: `https://ember.now-examples.now.sh`
```

This can be wrapped in backticks to make the linter happy.